pkg/catalog/versions/k8s.go

```diff
@@ -50,6 +50,12 @@ var kubernetesVersions = map[string]KubernetesVersions{
 		Etcd:       "3.5.12",
 		CoreDNS:    "current",
 	},
+	"1.31.0": {
```

Comment:
Should the entry be `1.31.5` instead? Refer https://github.com/oracle-cne/ock/pull/23/files#diff-6d5e9e7f8c42d3fedc073e2aa164dcd321f95cef2d7e6ea2ee74aa6f54adf1e4R2-R5

Reply:
The hotfix version is actually irrelevant.  The patches section (https://github.com/oracle-cne/ocne/blob/main/pkg/cluster/ignition/kubeadm.go#L32 and https://github.com/oracle-cne/ocne/blob/main/pkg/cluster/ignition/kubeadm.go#L138) combined with the upgrade script (https://github.com/oracle-cne/ock/blob/main/configs/common/systemd/ocne-kube-update.sh#L10) results in the node always forcing itself to use the versions that are already present on the system.